metricbeat/module/system/test_system.py

```diff
@@ -586,6 +586,7 @@ def test_socket_summary(self):
 
             assert isinstance(udp["all"]["count"], int)
 
+    @unittest.skipIf(sys.platform == "win32", "Flaky test")
```

Comment:
Flaky tests should have a open issue to address them in the future.

Reply:
I would have created one separately but I had around 40+ tests, it would have been hard to manage the tickets, I have taken screenshots with all the ones that kept on popping up

Reply:
Umm, does this test always fail in win32? If it always fail maybe instead of skipping it because of flakiness we can say in the comment that this test (or this feature) is not supported in win 32.

Reply:
the test does not always fail, it passed previously.

Reply:
If we plan to fix these flakiness at some moment we should have an issue for that, and reference it in the comment. Flaky tests can be hiding bugs.
But possibly it doesn't make much sense to dedicate a lot of effort to corner cases in 32-bits Windows :slightly_smiling_face: